superflore/generators/bitbake/ros_meta.py

```diff
@@ -42,9 +42,6 @@ def clean_ros_recipe_dirs(self, distro=None):
 
```

Comment:
Commit message: `don't generate` -> `Drop generation of`

Include the rationale from https://github.com/ros-infrastructure/superflore/issues/274 in the commit message body.